installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -37,6 +37,11 @@ include::modules/installation-osp-accessing-api.adoc[leveloffset=+1]
 include::modules/installation-osp-accessing-api-floating.adoc[leveloffset=+2]
 include::modules/installation-osp-accessing-api-no-floating.adoc[leveloffset=+2]
 include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
+include::modules/installation-osp-creating-network-resources.adoc[leveloffset=+1]
```

Comment:
This module shouldn't be moved, as it is modified already to update resources. But it will fail anyway as those resources was never created.

Just before module `installation-initializing.adoc` there should be a new one for creating network resources containing just one thing: https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md?plain=1#L273-L284 which means running `network.yaml` playbook.

Module `installation-osp-creating-network-resources.adoc` could be renamed to `installation-osp-update-network-resources.adoc` or similar.

Reply:
This makes a lot of sense now - I could not grasp why we were 'creating' network resources after installation! 

I have applied the changes, thanks for this crucial feedback.